ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/Apps.tsx

```diff
@@ -21,6 +27,13 @@ export function Apps() {
         {res.data.map((app) => {
           return <AppCard app={app} className="mb-4" envSlug={env.slug} key={app.id} />;
         })}
+        <Button
+          className="mx-auto"
+          kind="primary"
+          label="Sync New App"
+          btnAction={() => router.push(`/env/${env.slug}/apps/sync-new` as Route)}
```

Comment:
This can't be a nested route under `/apps` since it could conflict with a user's app name. If a user named their app `sync-new` then they'd never be able to navigate to its app page

Reply:
Ups, forgot to add a note to the PR! My idea is to create another PR on top of this to use a parent directory (.apps) and put them both on the same level. The reason I had decided not to do it already is because the directory change will create a lot of code conflicts with any open PRs you might have, and I didn't wanna mess with your code nor block myself.